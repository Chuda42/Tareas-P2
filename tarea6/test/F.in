hacerVecinos 1 2 5.0
hacerVecinos 1 3 5.0
hacerVecinos 1 4 5.0
hacerVecinos 1 5 5.0
hacerVecinos 1 6 5.0
hacerVecinos 1 7 5.0
hacerVecinos 1 8 5.0
hacerVecinos 1 9 5.0
hacerVecinos 1 10 5.0
hacerVecinos 10 2 1.0
hacerVecinos 10 3 1.0
hacerVecinos 10 4 1.0
hacerVecinos 10 5 1.0
hacerVecinos 10 6 1.0
hacerVecinos 10 7 1.0
hacerVecinos 10 8 1.0
hacerVecinos 10 9 1.0
distancia 5 1
distancia 5 10
sonVecinos 2 5
vecinos 10
vecinos 2
asociarEnMap 5 15.0
desasociarEnMap 5
asociarEnMap 5 25.0
desasociarEnMap 5
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
existeAsociacionEnMap 5
estaLlenoMap
actualizarEnCP 4 1.0
prioridad 3
prioritario
Fin
