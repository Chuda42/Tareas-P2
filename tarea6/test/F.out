1>Nuevos vecinos: 1 2 5.00.
2>Nuevos vecinos: 1 3 5.00.
3>Nuevos vecinos: 1 4 5.00.
4>Nuevos vecinos: 1 5 5.00.
5>Nuevos vecinos: 1 6 5.00.
6>Nuevos vecinos: 1 7 5.00.
7>Nuevos vecinos: 1 8 5.00.
8>Nuevos vecinos: 1 9 5.00.
9>Nuevos vecinos: 1 10 5.00.
10>Nuevos vecinos: 10 2 1.00.
11>Nuevos vecinos: 10 3 1.00.
12>Nuevos vecinos: 10 4 1.00.
13>Nuevos vecinos: 10 5 1.00.
14>Nuevos vecinos: 10 6 1.00.
15>Nuevos vecinos: 10 7 1.00.
16>Nuevos vecinos: 10 8 1.00.
17>Nuevos vecinos: 10 9 1.00.
18>5.00
19>1.00
20>NO son vecinos.
21>1 2 3 4 5 6 7 8 9 
22>1 10 
23>Establecida la asociación.
24>Eliminada la asociación.
25>Establecida la asociación.
26>Eliminada la asociación.
27>Se insertó el elemento.
28>Se insertó el elemento.
29>Se insertó el elemento.
30>No existe asociacion.
31>No lleno.
32>Se actualizó el valor.
33>3.00.
34>4
35>Fin.
